.cico/cico_functions.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 #
-# Copyright (c) 2012-2019 Red Hat, Inc.
+# Copyright (c) 2012-2020 Red Hat, Inc.
```

Comment:
has this really existed since 2012? I've been using 2018-2020 for most things...

Reply:
Huh, I just ran a find-replace. I wonder if we can track down the original `cico_functions.sh` -- it generally just gets copied in for everything that runs in centos CI.